diff --git a/node_modules/react-native-nfc-manager/ios/NfcManager.m b/node_modules/react-native-nfc-manager/ios/NfcManager.m
index 5869623..3a2219d 100755
--- a/node_modules/react-native-nfc-manager/ios/NfcManager.m
+++ b/node_modules/react-native-nfc-manager/ios/NfcManager.m
@@ -415,6 +415,7 @@ RCT_EXPORT_METHOD(getTag: (nonnull RCTResponseSenderBlock)callback)
             }
         } else {
             callback(@[@"No session available", [NSNull null]]);
+            return;
         }
         
         if (ndefTag) {
